### DISCUSSION TITLE
9.11 Resource Manager - Authorization has been denied for this request

### DISCUSSION THREAD

**Discussion by Xand0r** [dnnsoftware/Dnn.Platform#5608] [Q&A]
I've been using the Resource Manager module to display the files and folders for users that are not logged in. This functionality doesn't appear to exist in the new version of the Resource Manager module.

I've tested both the 9.11.0 update and the 9.11.1 update, but still get the error "Authorization has been denied for this request". And while the module itself does load, it displays this text, "Showing undefined of undefined items". 

Previously, if I had set the permissions correctly on the relevant folders and files within the Asset Manager, I could display files and folders for any user regardless of whether they are logged in or not.

![image](https://user-images.githubusercontent.com/1392486/230108511-a74fc134-595c-4188-9fe4-56c83e6f1a1d.png)

I guess I have two questions:

1) Is this functionality planned for the new Resource Manager? (I scoured the 9.11.0 and 9.11.1 Issues and Discussions, but didn't see anyone talking about this)

2) Is there a better module out there that I could use to display files/folders for "All Users"?

Thanks in advance for any advice you can offer!

**Comment by valadas** [comment]
This use case was simply not tested, I remember this was reported before but can't find an issue about it right now... I am not sure how easy/complex this would be to implement. If you need a quick solution I would look into another solution, possibly 2sxc or something like that.

**Reply by Xand0r** [reply]
Yes, I also thought I saw it listed as an Issue before as well. I think it might have gotten merged with another Resource Manager related Issue and then forgotten.

I appreciate your reply and I'll look into 2sxc. Thanks!

Should I promote this discussion to an Issue?

**Reply by valadas** [reply]
Sure
